StarCellBio/settings.py

```diff
@@ -26,6 +26,12 @@
     # ('Your Name', 'your_email@example.com'),
 )
 
+if os.uname()[1] != 'vagrant-ubuntu-trusty-64':
+    DEFAULT_FILE_STORAGE = 'libs.storages.S3Storage.S3Storage'
+    AWS_ACCESS_KEY_ID = ''
+    AWS_SECRET_ACCESS_KEY = ''
+    AWS_STORAGE_BUCKET_NAME = ''
```

Comment:
Rather than making a conditional based on the uname of the host I think it would be easier to use something like `S3_BACKEND_ENABLED = False` which can be overridden at deploy time.